### ISSUE TITLE
Port request: String Overrides

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/contrib#778]
**Name of the module, theme, or layout**
stringoverrides

**Link to the drupal.org module, theme, or layout**
https://www.drupal.org/project/stringoverrides

**Link to D7 source code**
https://git.drupalcode.org/project/stringoverrides/-/tree/7.x-1.x?ref_type=heads

**Jen's initial reaction: ~3 hrs**

- looks straightforward
- has tests (good!)
- drush integration → bee integration
- move some files around (theme functon → theme.inc file, css → css directory, submodule → modules directory)
- migrate submodule (NOT related to migrate module) - provides UI to import/export overrides (nice!) 

**(Optional) Link to an issue in the drupal.org queue asking the Drupal community if anyone is working on a Backdrop port of this project**
todo

**Comment by laryn** [comment]
@jenlampton I vaguely recall looking at this module at one point and then discovering it's already in core, in a way:
https://github.com/backdrop/backdrop/blob/1.x/settings.php#L299-L313

Of course, you may wish to port this module to make it available via the UI.

**Comment by jenlampton** [comment]
Yeah, I'd listed that as a possible alternative, bit I'm thinking we might
need the UI also.

**Comment by nattywebdev** [comment]
This would be an excellent addition to contrib modules - definitely gets my vote.
